Panic on Mac caused by cpu percentages issue

```
panic: runtime error: index out of range

goroutine 6 [running]:
github.com/cjbassi/gotop/widgets.(*CPU).update(0xc42000afc0)
	/home/cjbassi/go/src/github.com/cjbassi/gotop/widgets/cpu.go:52 +0x3c9
created by github.com/cjbassi/gotop/widgets.NewCPU
	/home/cjbassi/go/src/github.com/cjbassi/gotop/widgets/cpu.go:35 +0x2cc
```

Same. Running `macOS 10.12.6`.

```
panic: runtime error: index out of range

goroutine 6 [running]:
github.com/cjbassi/gotop/widgets.(*CPU).update(0xc42000b080)
	/home/cjbassi/go/src/github.com/cjbassi/gotop/widgets/cpu.go:52 +0x3c9
created by github.com/cjbassi/gotop/widgets.NewCPU
	/home/cjbassi/go/src/github.com/cjbassi/gotop/widgets/cpu.go:35 +0x2cc
```

Also on 10.12.6, clean install via 
```
$ go get -u -v github.com/cjbassi/gotop
$ gotop
```
and this is the error I'm getting after about 45-60 seconds:

```
Error caught. Exiting program.

Issue with CPU percentages.

Diagnostics:
self.Count: 4
gopsutil.Counts(): 4
len(percents): 0
percents: []
self.interval: 1s

panic: 1

goroutine 19 [running]:
github.com/cjbassi/gotop/utils.Error(0x1183b16, 0xf, 0xc42017e060, 0x52)
	/home/cjbassi/go/src/github.com/cjbassi/gotop/utils/utils.go:30 +0x23f
github.com/cjbassi/gotop/widgets.(*CPU).update(0xc420092fa0)
	/home/cjbassi/go/src/github.com/cjbassi/gotop/widgets/cpu.go:55 +0x532
created by github.com/cjbassi/gotop/widgets.NewCPU
	/home/cjbassi/go/src/github.com/cjbassi/gotop/widgets/cpu.go:37 +0x2d3
```

Have the same issue in Windows:
`Error caught. Exiting program.

Issue with CPU percentages.

Diagnostics:
self.Count: 8
gopsutil.Counts(): 8
len(percents): 0
percents: []
self.interval: 1s

panic: 1

goroutine 17 [running]:
github.com/cjbassi/gotop/utils.Error(0x5a120a, 0xf, 0xc0421720c0, 0x52)
        d:/WLCR/GoProjects/src/github.com/cjbassi/gotop/utils/utils.go:30 +0x246
github.com/cjbassi/gotop/widgets.(*CPU).update(0xc042140000)
        d:/WLCR/GoProjects/src/github.com/cjbassi/gotop/widgets/cpu.go:57 +0x539
created by github.com/cjbassi/gotop/widgets.NewCPU
        d:/WLCR/GoProjects/src/github.com/cjbassi/gotop/widgets/cpu.go:38 +0x2da`

Did by 
```
$ go get -u -v github.com/cjbassi/gotop
$ gotop
```

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -12,6 +12,7 @@
 builds:
   - binary: gotop
     goos:
+      - darwin
       - linux
     goarch:
       - amd64
@@ -27,3 +28,9 @@ archive:
   replacements:
     arm64: arm8
   format: tgz
+brew:
+  github:
+    owner: cjbassi
+    name: homebrew-gotop
+  description: "A terminal based graphical activity monitor inspired by gtop and vtop"
+  homepage: "https://github.com/cjbassi/gotop"

--- a/download.sh
+++ b/download.sh
@@ -12,6 +12,8 @@ download() {
 arch=$(uname -sm)
 case "$arch" in
     # order matters
+    Darwin\ *64)        download darwin_amd64   ;;
+    Darwin\ *86)        download darwin_386     ;;
     Linux\ armv5*)      download linux_arm5     ;;
     Linux\ armv6*)      download linux_arm6     ;;
     Linux\ armv7*)      download linux_arm7     ;;

--- a/main.go
+++ b/main.go
@@ -194,29 +194,29 @@ func initWidgets() {
 	wg.Add(widgetCount)
 
 	go func() {
-		defer wg.Done()
 		cpu = w.NewCPU(interval, zoom)
+		wg.Done()
 	}()
 	go func() {
-		defer wg.Done()
 		mem = w.NewMem(interval, zoom)
+		wg.Done()
 	}()
 	go func() {
-		defer wg.Done()
 		proc = w.NewProc(keyPressed)
+		wg.Done()
 	}()
 	if !minimal {
 		go func() {
-			defer wg.Done()
 			net = w.NewNet()
+			wg.Done()
 		}()
 		go func() {
-			defer wg.Done()
 			disk = w.NewDisk()
+			wg.Done()
 		}()
 		go func() {
-			defer wg.Done()
 			temp = w.NewTemp()
+			wg.Done()
 		}()
 	}
 

